src/config/logger.ts

```diff
@@ -43,45 +27,32 @@ const errorSerializer = (err) => err && {
 	request: requestSerializer(err.request)
 };
 
-const hashSerializer = (data: any): string | undefined => {
-	if (data === undefined || data == null) {
-		return undefined;
-	}
-	return createHashWithSharedSecret(data);
-};
-
-const sensitiveDataSerializers = (): Logger.Serializers => ({
-	jiraHost: hashSerializer,
-	orgName: hashSerializer,
-	repoName: hashSerializer,
-	userGroup: hashSerializer,
-	aaid: hashSerializer,
-	username: hashSerializer
-});
-
 const logLevel = process.env.LOG_LEVEL || "info";
 const globalLoggingLevel = levelFromName[logLevel] || INFO;
 
+const loggerStream = (isUnsafe) => ({
```

Comment:
Not really needed as a function, but needed to created 4 streams so made a little helper.
When probot goes will only have 2 streams so might remove it then.... maybe.

Reply:
sounds good, just add a typing to `isUnsafe` :)